setup.py

```diff
@@ -110,6 +110,27 @@
   DEFINE_MACROS += (('PyMODINIT_FUNC', pymodinit),)
 
 
+# By default, Python3 distutils enforces compatibility of
+# plugins with the OSX version Python3 was built with.
+# For Python3.4, this is OSX 10.6, but we need Thread Local Support (__thread)
+# from OSX 10.7
+def get_os_version_osx():
+  import sysconfig, re
+  platform = sysconfig.get_platform()
+  platform_match = re.match("macosx-(.*)-.*", platform)
```

Comment:
I got a bit confused initially because the variable is also named platform. We can use the platform module and it has a function `mac_ver` that returns a tuple:
```
>>> platform.mac_ver()
('10.11.4', ('', '', ''), 'x86_64')
```